.github/workflows/release-workflow.yml

```diff
@@ -82,13 +74,53 @@ jobs:
     - name: Build C Library
       run: |
         cmake --version
-        cmake -S . -B build
+        cmake -S . -B build -DBUILD_SHARED_LIBS=ON
         cmake --build build
-    - name: 'Build Simple Election'
-      run: | 
-        ElectionGuard_DIR="$PWD/build/ElectionGuard" cmake -S examples/simple -B simple_build
-        cmake --build simple_build --target simple
-    - name: 'Test Simple Election'
-      run: | 
-        cd simple_build
-        ./simple
\ No newline at end of file
+    - name: Upload Shared Library
+      uses: actions/upload-artifact@v1
+      with:
+        name: libelectionguard.dylib
+        path: build/libelectionguard.dylib
+  release:
+    name: Upload Release
+    needs: [linux_build, windows_build, macos_build]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1.0.0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          release_name: Milestone Release
+          tag_name: v1.0.0

```

Comment:
should this be a configurable var?  is every release going to have this same tag?  maybe I'm just not familiar with actions enough yet

Reply:
So ordinarily yes. This release tag is usually pulled in from the branches tag. However, this isn't being started by a branch but by a milestone so there's no associated branch tag. If you look at the release that is made, you'll notice the tag isn't set because there isn't an existing `v1.0.0` tag. I could try setting it to say `untagged` if you prefer? 

It's an oddity where the tag is required even if it's not used.

Reply:
im just thinking that we'll have to update this when we update the milestone.  would prefer to not do that, but it sounds like there isnt a good solution at the moment

Reply:
This issue was added to solve this: #28